src/mbgl/renderer/layers/render_circle_layer.hpp

```diff
@@ -34,9 +34,8 @@ class RenderCircleLayer: public RenderLayer {
     const style::CircleLayer::Impl& impl() const;
 };
 
-template <>
-inline bool RenderLayer::is<RenderCircleLayer>() const {
-    return type == style::LayerType::Circle;
+inline const RenderCircleLayer* toRenderCircleLayer(const RenderLayer* layer) {
+    return static_cast<const RenderCircleLayer*>(layer);
```

Comment:
can we just use the `static_cast` as is rather than having it in a separate function?

Reply:
Wrapper would allow us to add extra checks in one place. @pozdnyakov wdyt? Another option is to use wrapper similar to [chrome's](https://cs.chromium.org/chromium/src/third_party/blink/renderer/platform/wtf/casting.h?sq=package:chromium&g=0&l=101) casting that has asan checks.